csfieldguide/static/interactives/matrix-simplifier/js/matrix-simplifier.js

```diff
@@ -363,6 +392,10 @@ function showOutput() {
   var matrixRows = matrixToArray(matrix);
   var vectorRows = matrixToArray(vector);
 
+  // update global result variable
+  resultEqtn = matrixRows.toString() + ',v,' + vectorRows.toString();
```

Comment:
Consider a leading 'm' for the code, then when parsing the pasted content you could check for the components individually i.e. If string includes `v,` then what follows is the vector, if string includes `m,` then what follows is the matrix

My reasoning is that sometimes users may only want to copy over the matrix part of the code, as they have the vector set already or something